site/packages/aa-alchemy/provider/introduction.md

```diff
@@ -18,8 +18,8 @@ head:
 
 Notable differences between `AlchemyProvider` and `SmartAccountProvider` are implementations for:
 
-1.  [`gasEstimator`](/packages/aa-alchemy/provider/gasEstimator) -- overrides the `SmartAccountProvider` gas estimator.
-2.  [`withAlchemyGasManager`](/packages/aa-alchemy/provider/withAlchemyGasManager) -- adds the Alchemy Gas Manager middleware to the provider.
+1.  [`gasEstimator`](/packages/aa-alchemy/provider/withGasEstimator.md) -- overrides the base `SmartAccountProvider` gas estimator.
```

Comment:
I think this was right before. it's meant to say that the default `gasEstimator` middleware on `AlchemyProvider` is different from the default one